docs/source/release/v3.11.0/indirect_inelastic.rst

```diff
@@ -24,4 +24,7 @@ Improvements
 Bugfixes
 --------
 
+- An issue has been fixed in :ref:`algm-IndirectILLEnergyTransfer` when handling the data with mirror sense, that have shifted 0 monitor counts in the left and right wings.
+- An issue has been fixed in :ref:`algm-IndirectILLReductionFWS` when requesting the interpolation option for the detector calibration.
```

Comment:
With the release notes you mention when the issue occurs, but can you briefly clarify the what the issue is? E.g. in the first case an issue with different x-axes, the second an issue with the scale.